manifests/params.pp

```diff
@@ -149,7 +149,7 @@
   $autosignfile = "${puppetdir}/autosign.conf"
 
   # Template settings
-  $template_url = "http://${facts['networking']['fqdn']}:8000"
+  $template_url = "http://${facts['networking']['fqdn']}:${foreman_proxy::http_port}"
```

Comment:
This is invalid. At this point the class `foreman_proxy` is not initialized yet.
```suggestion
  $template_url = "http://${facts['networking']['fqdn']}:8000"
```